pkg/frontend/transport/handler.go

```diff
@@ -222,6 +228,17 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 		"split_queries", stats.LoadSplitQueries(),
 	}, formatQueryString(queryString)...)
 
+	if queryErr != nil {
+		logMessage = append(logMessage, []interface{}{
+			"status", "failed",
+			"error", queryErr,
```

Comment:
AFAIK, the Mimir convention is to use the label "err" when including errors in logs:
```suggestion
			"err", queryErr,
```